recipes/snakefmt/meta.yaml

```diff
@@ -1,20 +1,19 @@
-{% set name = "snakefmt" %}
```

Comment:
why remove this variable?

Reply:
Because we don't need it :).
`conda-skeleton` creates some redundant jinja templating of things that are actually static text for our recipe -- it only adds more clobber than anything. One example is that `name` variable which is static for `package: name:` and `source: url:`.